src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -199,8 +204,8 @@ export default {
 			// We don't want to open up $5 loan shares for loans with more than $25 at this time
 			// IF we wanted to show this interface on loans with less than 25 remaining they would see the selector
 			const minAmount = parseFloat(this.unreservedAmount < 25 ? this.minNoteSize : 25); // 25_hard_coded
-			// limit at 20 price options
-			const priceArray = buildPriceArray(parseFloat(this.unreservedAmount), minAmount).slice(0, 20);
+			// limit price options
+			const priceArray = this.enableFiveDollarsNotes ? build5DollarsPriceArray(parseFloat(this.unreservedAmount)).slice(0, 28) : buildPriceArray(parseFloat(this.unreservedAmount), minAmount).slice(0, 20); // eslint-disable-line max-len
```

Comment:
`parseFloat` can be moved out and stored in a different const to simplify this line of code.

Also, where does 28 come from, and I see that's different from the 20 options we had previously.

Reply:
Fixed with the function mentioned in the comment below.
Builders are returning all options from 5/25 to the amount left, so, if amount left is greater than 500 we are slicing the array to don't show more options than 500.
For $5 notes we are adding 8 options more than the current array.